### ISSUE TITLE
When the app is switched to the background, it is not locked

### ISSUE THREAD

**Issue by realkimmy** [neckaros/secure_application#21]
When the app is switched to the background, it is not locked

[repo](https://github.com/realkimmy/secure_app_bug_reporter)

**Comment by neckaros** [comment]
can you decribe the issue a little bit more? plarform/process...

**Comment by Zeroupper** [comment]
Yeah, this package didn't work for me as well. I tried to run it with the template code, and the locking didn't work.
